apps/site/pages/docs/getting-started/settingup-project.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: '2. Setting Up the Project'
+sidebar_label: 'Getting Started'
+---
+
+import { Callout } from 'nextra-theme-docs'
+import { Tab, Tabs } from 'nextra-theme-docs'
+
+<header>
+
+# Setting Up the Project
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+Now that your store is up and running, you're ready to set up your development environment and begin creating a unique storefront that reflects your brand. Let's get started!
+
+In this guide, you will:
+
+- Clone your store repository to your local machine.
+- Run a local development server at http://localhost:3000/.
+- Make your first change in your storefront.
+
+## Step 1: Prerequisites
+Before clonning the store's repository you will need to have the following
+tools in your machine:
+
+
+<details>
+  <summary>
+    <b>Node.js</b>
+  </summary>
+  <p>
+    <p>To run your FastStore project, you must have <b><a href="">Node.js</a></b> version 14.19 or later installed on your computer.</p>
```

Comment:
That's because Faststore CLI uses `AbortController`, which was introduced in Node v15.
```suggestion
    <p>To run your FastStore project, you must have <b><a href="">Node.js</a></b> version 15.0.0 or later installed on your computer.</p>
```